assets/migrations/scripts/20200630121930_add_column_in_task_metadata_table.sql

```diff
@@ -0,0 +1,14 @@
+-- // add column in task metadata table
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE core.task_metadata ADD COLUMN code VARCHAR DEFAULT NULL;
+
+CREATE INDEX task_metadata_index ON core.task_metadata(for_entity,plan_identifier,code);
```

Comment:
Add a query to populate code from existing tasks check this `20200310134153_add_owner_to_task_metadata.sql`

Reply:
Updated.